com.playeveryware.eos/Runtime/Core/Config/PlatformConfig.cs

```diff
@@ -26,17 +26,37 @@ namespace PlayEveryWare.EpicOnlineServices
     // referenced in the Epic namespace.
 #if !EOS_DISABLE
     using Epic.OnlineServices.IntegratedPlatform;
+    using Epic.OnlineServices.Auth;
+    using Epic.OnlineServices.Platform;
+    using Epic.OnlineServices.UI;
+    using UnityEditor.VersionControl;
```

Comment:
Why is version control necessary? Should this perhaps only be defined with UnityEditor?

Reply:
Nope. I've removed it.